lib/core/include/irods/irods_configuration_keywords.hpp

```diff
@@ -179,6 +179,12 @@ namespace irods
 
     extern const char* const KW_CFG_DEFAULT_RULE_ENGINE_PLUGIN_NAME;
     extern const char* const KW_CFG_DEFAULT_RULE_ENGINE_INSTANCE_NAME;
+
+    // access time
+    extern const char* const KW_CFG_ACCESS_TIME;
+    extern const char* const KW_CFG_ACCESS_TIME_QUEUE_NAME;
+    extern const char* const KW_CFG_ACCESS_TIME_QUEUE_SIZE;
+    extern const char* const KW_CFG_ACCESS_TIME_RESOLUTION_IN_SECONDS;
```

Comment:
Consider removing these or using them in the appropriate places.

Reply:
Updated one name. Decided to use them. Just makes it easy to update/change things if needed.